deployment/docker-compose/docker-compose.yml

```diff
@@ -7,14 +7,19 @@ services:
     command: >
       /bin/sh startup.sh
     restart: always
+    volumes:
+      - temp:/usr/src/aaq_backend/temp
```

Comment:
What is that for?

Reply:
that was for creating a temporary local copy of the `audio file` on our system for transcription purposes cause whisper needs the file locally somewhere for transcription, we later on delete it